charts/alertmanager/values.yaml

```diff
@@ -25,8 +25,6 @@ serviceAccount:
   # If not set and create is true, a name is generated using the fullname template
   name:
 
-podSecurityContext:
-  fsGroup: 65534
```

Comment:
@ybettan instead of removing, we should introduce `enabled` flag which would by default be `false`.

`podSecurityContext` section could be merged with `securityContext` with the flag.

Together they would look something like this:
```
securityContext:
    enabled: false
    fsGroup: 1001
    runAsNonRoot: true
    runAsUser: 1001
    runAsGroup: 1001
```

Reply:
Fixed, the object is slightly different as we are parsing it as a yaml so we can have all the fields in the same indention.